saleor/account/models.py

```diff
@@ -316,6 +333,10 @@ def can_login(self, site_settings: SiteSettings):
             or self.is_confirmed
         )
 
+    def save(self, *args, **kwargs):
```

Comment:
I believe it may not be sufficient to cover all possible customer updates. When using `update` or `bulk_update` in Saleor, the `save` function is not called automatically.

You should add/extend unit tests of any possible way of updating user `first_name` and `last_name` to check if `full_name` is updated.